lxd/scriptlet/starlark.go

```diff
@@ -140,13 +148,9 @@ func StarlarkMarshal(input any) (starlark.Value, error) {
 
 		sv = d
 	case reflect.Pointer:
-		if v.IsZero() {
```

Comment:
Is this part needed? Where is it accounted for now?

Reply:
This is handled by the `if input == nil` check on line 17. I moved this to the top of the function to avoid `nil` being mistaken for valid `starlark.Value`, which is just an unfortunate consequence of Go's type system and Starlark considering `nil` invalid.